### PR TITLE
Added textAttribute to the schema.yml

### PR DESCRIPTION
**Pull request by michael-r-may**
Adding the textAttribute to the yml get better swift code generation

### CODE REVIEW

**Comment by pjlee11** [comment]
Blocked on in-flight https://github.com/BBC-News/simorgh/issues/404 otherwise the `schema.yaml` will have incorrect definitions of `array`'s as `object`'s

File: data/schema.yaml [hunk 1]
```diff
@@ -228,6 +228,18 @@ components:
...
 
+    textAttribute:
+      enum:
+        - italic
+        - bold
+      example: italic
+      type: string
+
+    textAttributes:
+      type: array
```
Comment (pjlee11) [inline data/schema.yaml:287]:
```suggestion
      type: array
      uniqueItems: true
```

File: data/schema.yaml [hunk 2]
```diff
@@ -347,7 +359,9 @@ components:
             text:
               type: string
             attributes:
-              type: object
+              type: array
```
Comment (pjlee11) [inline data/schema.yaml:410]:
```suggestion
              type: array
              uniqueItems: true
```

File: data/schema.yaml [hunk 3]
```diff
@@ -228,6 +228,18 @@ components:
...
     # schema definitions for referred blocks below
 
+    textAttribute:
+      enum:
+        - italic
+        - bold
+      example: italic
+      type: string
+
+    textAttributes:
```
Comment (pjlee11) [inline data/schema.yaml:286]: A comment to clarify the use case for this in Apps would be good as currently this appears to be orphaned

Comment (michael-r-may) [reply data/schema.yaml:286]: 👍

**Comment by pjlee11** [comment]
Closing in favour of https://github.com/BBC-News/simorgh/pull/985

**Pull request closed without merging**